docs/learn/learn-account-multisig.md

```diff
@@ -9,148 +9,116 @@ slug: ../learn-account-multisig
+
+### Example using Multisig Accounts
 
-Currently, the **DepositBase** equals `deposit(1, 88)` (key size is 32; value is size 4+4+16+32 = 56
-bytes) and the **DepositFactor** equals `deposit(0, 32)` (additional address of 32 bytes).
+:::info Walk-through video tutorial
 
-The deposit function in JavaScript is defined below, cribbed from the
-[Rust source](https://github.com/paritytech/polkadot/blob/master/runtime/polkadot/constants/src/lib.rs).
+You can also see [this video tutorial](https://www.youtube.com/watch?v=-cPiKMslZqI) for more information about transacting with multisigs using the Accounts Tab, or [this other video](https://www.youtube.com/watch?v=T0vIuJcTJeQ) using the Extrinsic Tab in the [Polkadot-JS UI](https://polkadot.js.org/apps/#/accounts).
 
-```js
-// Polkadot
-const DOLLARS = 10000000000; // planck
-const MILLICENTS = 100000; // planck
+:::
 
-// Kusama
-// const DOLLARS = 166666666666.67;
-// const MILLICENTS = 1666666.66;
+![multisig diagram](../assets/multisig-diagram.png)
 
-const deposit = (items, bytes) => {
-  return items * 20 * DOLLARS + bytes * 100 * MILLICENTS;
-};
+Let's consider an example of a multisig on Polkadot with a threshold of 2 and 3 signers: Charlie,
+Dan, and Eleanor. First, Charlie will create the call on-chain by calling the `multisig.asMulti` extrinsic with the raw
+call, in this case, a balance transfer (`balances.transferKeepAlive` extrinsic) from multisig CDE to Frank's account. When doing this, Charlie will have to deposit `DepositBase + (2 * DepositFactor) = 20.152 DOT`
+while he waits for either Dan or Eleanor also to approve the balance transfer call using the `multisig.approveAsMulti` or the `multisig.asMulti` extrinsics.
 
-console.log("DepositBase", deposit(1, 88));
-console.log("DepositFactor", deposit(0, 32));
-```
+If Dan submits the `multisig.approveAsMulti` extrinsic, he approves Charlie's call but he passes on the final approval to Eleanor. So, although the multisig has threshold 2, in this case all 3/3 signatories need to participate in the transaction approval. Eleanor will need to submit a `multisig.asMulti` or `multisig.approveAsMulti` extrinsic to transfer funds from CDE to Frank.
+
+Alternatively, Dan or Eleanor can just submit a `multisig.asMulti` extrinsic after Charlie to transfer the funds. In this case, 2/3 signatories will participate in the transaction approval. The accounts approving Charlie's call will not need to place the deposit, and Charlie will receive his
+deposit back once the transfer is successful or canceled. To cancel the transaction, Dan or Eleanor can use the `multisig.cancelAsMulti` extrinsic.
+
+Note that multisigs are **deterministic**, which means that multisig addresses are generated from the addresses of signers and threshold of the multisig wallet. No matter the order of the signatories' accounts the multisig will have always the same address because accounts' addresses are sorted in ascending order.
```

Comment:
```suggestion
Note that multisigs are **deterministic**, which means that multisig addresses are generated from the addresses of signers and the threshold of the multisig wallet. No matter the order of the signatories' accounts, the multisig will always have the same address because accounts' addresses are sorted in ascending order.
```